sds_data_manager/lambda_code/SDSCode/indexer.py

```diff
@@ -180,11 +180,8 @@ def lambda_handler(event, context):
     logger.info("Allowed file types: " + str(filetypes))
 
     # Grab environment variables
-    host = os.environ["OS_DOMAIN"]
-    snapshot_repo_name = os.environ["SNAPSHOT_REPO_NAME"]
-    snapshot_s3_bucket = os.environ["S3_SNAPSHOT_BUCKET_NAME"]
-    snapshot_role_arn = os.environ["SNAPSHOT_ROLE_ARN"]
-    region = os.environ["REGION"]
+    os.environ["OS_DOMAIN"]
+    os.environ["REGION"]
 
     # create opensearch client
     client = _create_open_search_client()
```

Comment:
Are those going to get removed in a separate PR? just curious. And separate PR might be easier and it was lot easier to remove :)

Reply:
Yeah, I'm going to remove all the OpenSearch code in a separate PR once RDS is developed enough to fully replace it.